modules/data.atmosphere/R/download.GLDAS.R

```diff
@@ -0,0 +1,156 @@
+##' Download and convert single grid point GLDAS to CF single grid point from hydro1.sci.gsfc.nasa.gov using OPENDAP interface
+##' @name download.GLDAS
+##' @title download.GLDAS
+##' @export
+##' @param outfolder
+##' @param start_date
+##' @param end_date
+##' @param site_id
+##' @param lat
+##' @param lon
+##'
+##' @author Christy Rollinson
+
+download.GLDAS <- function(outfolder, start_date, end_date, site_id, lat.in, lon.in, overwrite=FALSE, verbose=FALSE, ...){  
+  require(PEcAn.utils)
+  require(RCurl)
+  require(XML)
+  require(lubridate)
+  require(ncdf4)
+  require(stringr)
```

Comment:
The code below will not run (I assume) without these packages; use `library()`, not `require()`.